app/src/main/res/layout/contributors_activity.xml

```diff
@@ -39,15 +47,34 @@
                         app:layout_collapseMode="parallax"
                         app:layout_collapseParallaxMultiplier="0.7">
 
-                        <ImageView
+                        <FrameLayout
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
-                            android:layout_gravity="center_horizontal"
-                            android:layout_marginTop="16dp"
-                            android:scaleType="center"
-                            android:scaleX="0.8"
-                            android:scaleY="0.8"
-                            android:src="@mipmap/ic_splash" />
+                            android:layout_gravity="center"
+                            android:layout_marginTop="25dp"
+                            android:layout_marginBottom="25dp">
```

Comment:
The image had 16dp top margin previously, please leave it this way. If you need more spacing for `@+id/glove`, add it to that element instead - this way it won't affect normal user's experience.